src/main.cpp

```diff
@@ -1,39 +1,55 @@
-#include <iostream>
-
-#include "ftxui/dom/elements.hpp"
-#include "ftxui/screen/screen.hpp"
-#include "ftxui/screen/string.hpp"
-
-int main(void) {
-  using namespace ftxui;
-
-  auto summary = [&] {
-    auto content = vbox({
-        hbox({text(L"- done:   "), text(L"3") | bold}) | color(Color::Green),
-        hbox({text(L"- active: "), text(L"2") | bold}) | color(Color::RedLight),
-        hbox({text(L"- queue:  "), text(L"9") | bold}) | color(Color::Red),
-    });
-    return window(text(L" Summary "), content);
-  };
-
-  auto document =  //
-      vbox({
-          hbox({
-              summary(),
-              summary(),
-              summary() | flex,
-          }),
-          summary(),
-          summary(),
-      });
-
-  // Limit the size of the document to 80 char.
-  document = document | size(WIDTH, LESS_THAN, 80);
-
-  auto screen = Screen::Create(Dimension::Full(), Dimension::Fit(document));
-  Render(screen, document);
-
-  std::cout << screen.ToString() << std::endl;
-
-  return EXIT_SUCCESS;
+#include <memory>  // for allocator, __shared_ptr_access
```

Comment:
This change seems unrelated to your patch. Could you revert it?

Reply:
I had to include this change since the previous example wasn't working for an unknown reason in wasm. I suppose this is due to the fact that we aren't in a loop and thus it hasn't time to process the render before it exits. 
Maybe this is just an issue on my machine, if you could try it yourself and let me know i would appreciate